src/components/pages/HomePage.js

```diff
@@ -41,6 +41,7 @@ class HomePage extends React.Component {
           <h2>Stonehaven Academy</h2>
           {/* <img id="path-image" src="./images/home-path-image.jpg" alt="Web Development Image" /> */}
         </div>
+        <hr className="homepage-line" />
```

Comment:
Just so it's styled and named in the same way as the other lines on the site.

```suggestion
        <hr className="breakline" />
```